ofono/doc/connman-api.txt

```diff
@@ -155,6 +155,13 @@ Methods		dict GetProperties()
 					 [service].Error.AttachInProgress
 					 [service].Error.NotImplemented
 
+Methods		void ProvisionContext()
+			Resets all properties back to default. Fails if
+			context is being activated or deactivated.
+
+			Possible Errors: [service].Error.Failed
+					 [service].Error.InProgress
+
```

Comment:
Please double check the possible errors, there's e.g. __ofono_error_not_available